docs/contribute/processes/continuous-integration.md

```diff
@@ -13,8 +13,8 @@ The results of the analysis are available through the GitHub interface or can be
 
 ## Pull-Request-Triggered Jobs
 
-Travis CI also runs when a new pull request is created or an existing pull request is updated.
-See the [Travis documentation](https://docs.travis-ci.com/user/pull-requests/#%E2%80%98Double-builds%E2%80%99-on-pull-requests) for more information.
+GitHub Actions also runs when a new pull request is created or an existing pull request is updated.
```

Comment:
There are still more references to Travis CI builds that need to be updated in the section `Push-Triggered Jobs`.

It is also mentioned in the section `Daily Jobs`, but, As the CRON job would still be active after this, such references can remain as far as I can tell.